InstanceManager.js

```diff
@@ -6,7 +6,19 @@ const LocalRunner = require('./LocalRunner.js');
 const DockerRunner = require('./DockerRunner.js');
 const common = require('./common.js');
 const endpointToUrl = common.endpointToUrl;
+const ERR_IN_SHUTDOWN = 30;
 const WAIT_TIMEOUT = 400; // seconds
+const FailoverError = require('./Errors.js').FailoverError;
+const debugLog = (logLine) => {
+  if (process.env.LOG_IMMEDIATE && process.env.LOG_IMMEDIATE == "1") {
+    console.log((new Date()).toISOString(), logLine);
+  };
+};
+
+const shutdownKillAttempts = 3600;  // 180s, after this time we kill the instance
+const shutdownMaxAttempts = 4000;   // 200s, note that the cluster internally
+                                    // has a 120s timeout
```

Comment:
Are these comments lying or are the numbers in different units? Seems odd to specify the behaviour in seconds in the comment and then define it in terms of number of attempts.